sbt-crossproject/src/main/scala/sbtcrossproject/package.scala

```diff
@@ -1,4 +1,15 @@
+import scala.language.experimental.macros
+
 import sbt._
-import scala.language.implicitConversions
 
-package object sbtcrossproject extends CrossProjectExtra
+package object sbtcrossproject {
```

Comment:
It looks like `import sbt` above is unused.

Reply:
Indeed. Fixed.